examples/guestbook/redis-slave/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=redis:3.2.9
+arm=arm32v6/redis:3.2.9-alpine
```

Comment:
Because I see only alpine images there in the hub.docker.com, not sure why they are not building with normal distro!